### PR TITLE
[HITL] Add task config for single-learn data collection

### PR DESCRIPTION
**Pull request by Ram81**
## Motivation and Context

Add task config for multi-agent data collection using spot and humanoid

## How Has This Been Tested

```
python examples/hitl/rearrange_v2/rearrange_v2.py --config-name lang_rearrange_spot_humanoid
````

## Types of changes

HITL 

## Checklist

- [ ] My code follows the code style of this project.
- [ ] I have updated the documentation if required.
- [ ] I have read the [**CONTRIBUTING**](/CONTRIBUTING.md) document.
- [ ] I have completed my CLA (see **CONTRIBUTING**)
- [ ] I have added tests to cover my changes if required.

### CODE REVIEW

File: examples/hitl/rearrange_v2/config/language_rearrange_multi_agent.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  - /habitat/task/actions@habitat.task.actions.agent_1_arm_action: arm_action
+  - /habitat/task/actions@habitat.task.actions.agent_1_base_velocity: base_velocity
+  - /habitat/task/actions@habitat.task.actions.agent_1_rearrange_stop: rearrange_stop
+  - _self_
+
+habitat:
+  task:
+    actions:
+      agent_1_arm_action:
+        grip_controller: MagicGraspAction
```
Comment (0mdc) [inline examples/hitl/rearrange_v2/config/language_rearrange_multi_agent.yaml:17]: This action interferes with the HITL application. It must be removed.

File: examples/hitl/rearrange_v2/rearrange_v2.py [hunk 2]
```diff
@@ -32,6 +32,11 @@
 from habitat_hitl.environment.hablab_utils import get_agent_art_obj_transform
 from habitat_sim.utils.common import quat_from_magnum, quat_to_coeffs
 
+try:
+    from habitat_llm.agent.env import dataset  # noqa: F401
+except ImportError:
+    print("Habitat-LLM not installed. Skipping import.")
+
```
Comment (0mdc) [inline examples/hitl/rearrange_v2/rearrange_v2.py:39]: This is now in main [here](https://github.com/facebookresearch/habitat-lab/pull/1933).

File: habitat-hitl/habitat_hitl/environment/controllers/controller_helper.py [hunk 3]
```diff
@@ -134,7 +134,9 @@ def __init__(
                         # sloppy: derive turn scale. This is the change in yaw (in radians) corresponding to a base ang vel action of 1.0. See also Habitat-lab BaseVelAction.
                         turn_scale = (
                             config.habitat.simulator.ctrl_freq
-                            / config.habitat.task.actions.agent_0_base_velocity.ang_speed
+                            / config.habitat.task.actions[
+                                f"{agent_k}_base_velocity"
+                            ].ang_speed
```
Comment (0mdc) [inline habitat-hitl/habitat_hitl/environment/controllers/controller_helper.py:139]: 👍

File: examples/hitl/rearrange_v2/config/lang_rearrange_spot_humanoid.yaml [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+  - hitl_defaults
+  - _self_
+
+habitat:
+  # various config args to ensure the episode never ends
+  environment:
+    max_episode_steps: 0
+    iterator_options:
+      # For the demo, we want to showcase the episodes in the specified order
+      shuffle: False
```
Comment (0mdc) [inline examples/hitl/rearrange_v2/config/lang_rearrange_spot_humanoid.yaml:14]: More attributes must be added here for the desired outcome.

See [this PR](https://github.com/facebookresearch/habitat-lab/pull/1935).

File: examples/hitl/rearrange_v2/config/language_rearrange.yaml [hunk 5]
```diff
@@ -43,23 +43,17 @@ habitat:
     task_spec: pddl/multi_agent_tidy_house
     pddl_domain_def: fp
     actions:
-      base_velocity:
+      agent_0_base_velocity:
```
Comment (0mdc) [inline examples/hitl/rearrange_v2/config/language_rearrange.yaml:46]: Should be uniformized with the other velocities in this config.

I deployed the following:
```
lin_speed: 10.0
ang_speed: 300
```

File: examples/hitl/rearrange_v2/config/language_rearrange_multi_agent.yaml [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+      agent_1_base_velocity:
+        lin_speed: 40.0
+        ang_speed: 20.0
```
Comment (0mdc) [inline examples/hitl/rearrange_v2/config/language_rearrange_multi_agent.yaml:20]: Same comment as above.

File: examples/hitl/rearrange_v2/config/language_rearrange_multi_agent.yaml [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+  gym:
+    obs_keys:
+      - agent_0_head_depth
+      - agent_0_is_holding
+      - agent_0_ee_pos
+      - agent_1_head_depth
```
Comment (0mdc) [inline examples/hitl/rearrange_v2/config/language_rearrange_multi_agent.yaml:26]: @zephirefaith Do we need all these sensors?

**Review by 0mdc — COMMENTED** [review]
LGTM! I've been using these configs for a while, and they are deployed.

@zephirefaith : The comments here are probably relevant to you.

File: examples/hitl/rearrange_v2/config/language_rearrange_multi_agent.yaml (hunk 7 above)
Comment (zephirefaith) [reply examples/hitl/rearrange_v2/config/language_rearrange_multi_agent.yaml:26]: We need the panoptics, we do not need RGB or depth. I can remove those.

Comment (0mdc) [reply examples/hitl/rearrange_v2/config/language_rearrange_multi_agent.yaml:26]: We also need to make sure that the sensors don't run in single-user and multi-user as these applications run on servers without GPUs.

**Review by 0mdc — APPROVED** [review]
Thanks for the changes! LGTM!

**Pull request merged**